src/cc2olx/tools/video_upload.py

```diff
@@ -0,0 +1,253 @@
+import argparse
+import csv
+import os
+from pathlib import Path
+
+import requests
+from requests.auth import AuthBase
+
+
+OAUTH_TOKEN_URL = 'https://courses.edx.org/oauth2/access_token'
```

Comment:
We wanted to avoid having a full fledged Django application, which is why these are not in settings.